BDArmory/Distribution/GameData/BDArmory/Parts/ArmorPlate/TripanelArmor.cfg

```diff
@@ -9,22 +9,11 @@ buoyancy = 1
 		model = BDArmory/Parts/ArmorPlate/TriPanel
 		scale = 1.0, 1.0, 1.0
 	}
-	NODE
-	{
-		name = Node1
-		transform = N1
-		size = 0
-		method = FIXED_JOINT
-	}
-	NODE
-	{
-		name = Node2
-		transform = N2
-		size = 0
-		method = FIXED_JOINT
-	}
 rescaleFactor = 1
```

Comment:
Hopefully this shouldn't break existing craft using triangular panels too badly...